### DISCUSSION TITLE
Spectrum 2 launch date

### DISCUSSION THREAD

**Discussion by joebudi** [adobe/spectrum-css#3411] [General]
Hi! I was wondering when Spectrum 2 will be available to the public?

**Comment by castastrophe** [comment]
It's currently being released under our "next" tag but is not yet ready for production.

**Reply by joebudi** [reply]
Thank you for all your hard work. Is there a migration guide from s1 to s2?

**Reply by castastrophe** [reply]
We're working hard on one. The changelogs for each component do outline the updates as clearly as we were able if you want to start there. The biggest shift will be in the global token data which has undergone a major rework and has lots of breaking changes in it (@spectrum-css/tokens).
